File: src/Unidesk/Unidesk/Dtos/SimpleJsonResponse.cs
using FluentValidation.Results;

namespace Unidesk.Dtos;

public class SimpleJsonResponse : DtoBase
{
    public bool Success { get; set; } = true;
    public string Message { get; set; }

    public IEnumerable<string> StackTrace { get; set; } = Enumerable.Empty<string>();
    public string? DebugMessage { get; set; }
    public IEnumerable<ValidationFailure> Errors { get; set; }
}

File: src/Unidesk/Unidesk.Client/src/api-client/models/SimpleJsonResponse.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ValidationFailure } from './ValidationFailure';

export type SimpleJsonResponse = {
    success?: boolean;
    message?: string | null;
    stackTrace?: Array<string> | null;
    debugMessage?: string | null;
    errors?: Array<ValidationFailure> | null;
};
